stacks/spica/src/bucket/pages/bucket-add/bucket-add.component.html

```diff
@@ -1,504 +1,397 @@
-      mat-raised-button
-      color="warn"
-      *matSave="$remove | async; let state"
-      [disabled]="!!state"
-      [ngSwitch]="state"
-    >
-      <ng-container *ngSwitchDefault>
-        <mat-icon>delete</mat-icon>
-        <span>Remove</span>
-      </ng-container>
-      <ng-container *ngSwitchCase="'saving'">
+                  <mat-icon>{{ _inputResolver.resolve(propertyKv.value.type).icon }}</mat-icon>
+                  <span>{{ propertyKv.value.title }}</span>
+                </mat-card>
+                <span>Left</span>
+              </mat-grid-tile>
+              <mat-grid-tile
+                [colspan]="2"
+                #rightList="cdkDropList"
+                cdkDropList
+                cdkDropListSortingDisabled
+                [cdkDropListData]="propertyPositionMap.right"
+                [cdkDropListConnectedTo]="[leftList, bottomList, nonPositioned]"
+                (cdkDropListDropped)="setPosition($event, 'right')"
+              >
+                <mat-card
+                  class="mat-elevation-z0"
+                  *ngFor="let propertyKv of propertyPositionMap.right"
+                  cdkDrag
+                >
+                  <mat-icon>{{ _inputResolver.resolve(propertyKv.value.type).icon }}</mat-icon>
+                  <span>{{ propertyKv.value.title }}</span>
+                </mat-card>
+                <span>Right</span>
+              </mat-grid-tile>
+              <mat-grid-tile
+                [colspan]="4"
+                #bottomList="cdkDropList"
+                cdkDropList
+                cdkDropListSortingDisabled
+                [cdkDropListData]="propertyPositionMap.bottom"
+                [cdkDropListConnectedTo]="[rightList, leftList, nonPositioned]"
+                (cdkDropListDropped)="setPosition($event, 'bottom')"
+              >
+                <mat-card
+                  class="mat-elevation-z0"
+                  *ngFor="let propertyKv of propertyPositionMap.bottom"
+                  cdkDrag
+                >
+                  <mat-icon>{{ _inputResolver.resolve(propertyKv.value.type).icon }}</mat-icon>
+                  <span>{{ propertyKv.value.title }}</span>
+                </mat-card>
+                <span>Bottom</span>
+              </mat-grid-tile>
+            </mat-grid-list>
+          </mat-list-item>
+        </div>
+      </mat-list>
+    </mat-card-content>
+    <mat-card-actions>
+      <button
+        class="save-button"
+        mat-button
+        [canInteract]="bucket._id ? 'bucket:update' : 'bucket:create'"
+        [resource]="bucket._id ? bucket._id : undefined"
+        form="describeForm"
+        *matSave="$save | async; let state"
+        [disabled]="
+          describeForm.invalid ||
+          (bucket.properties | keyvalue)?.length < 1 ||
+          !isThereVisible ||
+          bucket.primary == undefined ||
+          !!state
+        "
+        (click)="saveBucket()"
+        [ngSwitch]="state"
+      >
```

Comment:
Could you please check this button's css? It needs to more margin from right and top.